tests/test_orient_symmetric.py

```diff
@@ -122,7 +136,12 @@ def test_estimate_rotations(n_img, L, order, dtype):
         )
 
     # Assert mean angular distance is reasonable.
-    if order > 4:
+    if order == 2:
+        # For small n-img, we get some bad estimates so checking the
```

Comment:
Why not test with more images then?

Reply:
Good point. Bumped the number images from 45 to 55 and was able to get rotation estimates with 4 degrees on average. Added about 15 secs to the test suite.

Reply:
Sounds good. Let's do that then.

Reply:
Done